object/link.go

```diff
@@ -0,0 +1,111 @@
+package object
+
+import (
+	v2object "github.com/nspcc-dev/neofs-api-go/v2/object"
+	"github.com/nspcc-dev/neofs-api-go/v2/refs"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+)
+
+// Link represents a helper object with information about
+// a certain split chain of the NeoFS object. It is compatible with
+// NeoFS API V2 protocol.
+//
+// Link instance can be written to the [Object], see
+// [Object.WriteLink]/[Object.ReadLink].
+type Link v2object.Link
+
+// WriteLink writes [Link] to the [Object], and sets its type to [TypeLink].
```

Comment:
`[...]` notations for prms are redundant: they all are links already (e.g. https://pkg.go.dev/github.com/nspcc-dev/neofs-sdk-go@v1.0.0-rc.11/object#Object.WriteLock)

Reply:
dropped but i cannot understand why it is a problem. i literally copied the Lock docs (like from the link), what should your link say to me?

Reply:
I'd rather have `[Link]`, but if it's autodetected for these cases then it's not an error at least to not have brackets.